.github/workflows/ci.yml

```diff
@@ -484,11 +484,25 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
+        java: [temurin@11]
     runs-on: ${{ matrix.os }}
     steps:
       - name: Checkout current branch (fast)
         uses: actions/checkout@v3
 
+      - name: Setup Java (temurin@17)
```

Comment:
Actually there was a mistake here before I realized I could delete it. The matrix specifies 11 but the generated steps set up 17 ...

Reply:
Updated